src/events/gamebanana_watcher.js

```diff
@@ -0,0 +1,144 @@
+require('dotenv').config();
+const { MessageEmbed } = require('discord.js');
+const fetch = require('node-fetch');
+
+module.exports = {
+    name: 'ready',
+    execute: async (client) => {
+        let cssMaps = [];
+        let csgoMaps = [];
```

Comment:
No need to create 2 separate arrays here instead use 1 array and store mod ID to avoid duplicates.